src/ray/core_worker/fiber.h

```diff
@@ -83,6 +83,12 @@ using FiberChannel = boost::fibers::unbuffered_channel<std::function<void()>>;
 
 class FiberState {
  public:
+  static bool NeedDefaultExecutor(int32_t max_concurrency_in_default_group) {
+    RAY_UNUSED(max_concurrency_in_default_group);
```

Comment:
Can we just do RAY_CHECK(max_concurrency_in_default_group == 1)?

Reply:
The max_concurrency of default group may >= 1.